app/src/main/java/com/bigyoshi/qrhunt/qr/FragmentQrProfile.java

```diff
@@ -27,18 +29,20 @@
 
     private int pos;
     private PlayableQrCode removeQR;
-    private Player playerInfo;
+    private Player viewingPlayer;
+    private Player owner;
 
     /**
      * Constructor method
      *  @param i int
      * @param currentQR QR to remove
-     * @param playerInfo
+     * @param viewingPlayer
      */
-    public FragmentQrProfile(int i, PlayableQrCode currentQR, Player playerInfo) {
+    public FragmentQrProfile(int i, PlayableQrCode currentQR, Player viewingPlayer, Player owner) {
         this.pos = pos;
         this.removeQR = currentQR;
-        this.playerInfo = playerInfo;
+        this.viewingPlayer = viewingPlayer;
+        this.owner = owner;
```

Comment:
I don't understand what the distinction/point of having these 2 seperate things are for

Reply:
That for when we added the functionality to view other people's profiles, Don't want to give them the ability to delete qr codes on a profile that isn't theirs

Reply:
we only need the viewing player. PlayableQrCode will have playerId property which defines the owner of the QR code already

Reply:
My bad, completely forgot that the PlayableQRCode has the playerId saved

Reply:
WIll remove